.travis.yml

```diff
@@ -1,10 +0,0 @@
-language: java
-jdk:
-  - openjdk7
```

Comment:
This line here leads me to believe that it is not being used at all since master has long required JDK 8.